biggraphite/drivers/elasticsearch.py

```diff
@@ -286,20 +285,116 @@ def delete_directory(self, name):
         self._directory_names.remove(name)
 
     @staticmethod
```

Comment:
if this are static methods, moving them outside of the class might make them easier to read /test

Reply:
Done